### PR TITLE
feat: banner component

### PR DESCRIPTION
**Pull request by harshitadaswani**
- added banner component of landing page

### CODE REVIEW

File: src/pages/home/Banner.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+                        <p className="m-xs p-xs">
+                            Creators from all across the globe. Whether you are interested in digital art, photography or unique
+                            mixed media, you can find it here.
+                        </p>
+
+                        <div className="txt-center">
+                            <button className="btn">
+                                Learn More
+                                <i className="fas fa-arrow-right"></i>
```
Comment (rkapoor10) [inline src/pages/home/Banner.jsx]: you can use react icons here

**Review by rkapoor10 — APPROVED** [review]
The code is well structured and looks good to merge...go through the suggestion.

File: src/pages/home/Banner.jsx (hunk 1 above)
Comment (harshitadaswani) [reply src/pages/home/Banner.jsx]: done!

**Review by JantuDeb — APPROVED** [review]
Code looks fine. I think you can add more than one component if there's fewer lines of code.

**Pull request merged**